src/Umbraco.Web.Common/Security/MembersUserManager.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Principal;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Umbraco.Core;
+using Umbraco.Core.Configuration.Models;
+using Umbraco.Core.Models.Membership;
+using Umbraco.Core.Security;
+using Umbraco.Extensions;
+using Umbraco.Infrastructure.Security;
+using Umbraco.Net;
+using Umbraco.Web.Models.ContentEditing;
+
+
+namespace Umbraco.Web.Common.Security
+{
+    public class MembersUserManager : UmbracoUserManager<MembersIdentityUser, MemberPasswordConfigurationSettings>, IMembersUserManager
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public MembersUserManager(
+            IIpResolver ipResolver,
+            IUserStore<MembersIdentityUser> store,
+            IOptions<MembersIdentityOptions> optionsAccessor,
+            IPasswordHasher<MembersIdentityUser> passwordHasher,
+            IEnumerable<IUserValidator<MembersIdentityUser>> userValidators,
+            IEnumerable<IPasswordValidator<MembersIdentityUser>> passwordValidators,
+            //TODO: do we need members versions of this?
+            BackOfficeLookupNormalizer keyNormalizer,
+            BackOfficeIdentityErrorDescriber errors,
+            IServiceProvider services,
+            IHttpContextAccessor httpContextAccessor,
+            ILogger<UserManager<MembersIdentityUser>> logger,
+            IOptions<MemberPasswordConfigurationSettings> passwordConfiguration)
+            : base(ipResolver, store, optionsAccessor, passwordHasher, userValidators, passwordValidators, keyNormalizer, errors, services, logger, passwordConfiguration)
+        {
+            _httpContextAccessor = httpContextAccessor;
+        }
+        private string GetCurrentUserId(IPrincipal currentUser)
+        {
+            UmbracoBackOfficeIdentity umbIdentity = currentUser?.GetUmbracoIdentity();
+            var currentUserId = umbIdentity?.GetUserId<string>() ?? Core.Constants.Security.SuperUserIdAsString;
+            return currentUserId;
+        }
+
+        private IdentityAuditEventArgs CreateArgs(AuditEvent auditEvent, IPrincipal currentUser, string affectedUserId, string affectedUsername)
+        {
+            var currentUserId = GetCurrentUserId(currentUser);
+            var ip = IpResolver.GetCurrentRequestIpAddress();
+            return new IdentityAuditEventArgs(auditEvent, ip, currentUserId, string.Empty, affectedUserId, affectedUsername);
+        }
+
+
+        // TODO: As per backoffice, review where these are raised and see if they can be simplified and either done in the this usermanager or the signin manager,
```

Comment:
For members we don't really have this type of auditing so it's not required. Lets remove this for now and revisit later if we need any auditing on member info at the level of users. Also since the user events will be entirely refactored to use IEventAggregator anyways.

Reply:
OK, removing all member auditing events for now. However this still leaves the following from the `IUmbracoUserManager` interface, although there is a comment saying we will revisit these anyway. I'll leave them in for now, unimplemented, until we do that:

![image](https://user-images.githubusercontent.com/1649855/104188555-527c7f00-5411-11eb-86b4-2767cdd97f0d.png)